README.md

```diff
@@ -1,9 +1,54 @@
-# pm-idm
+# IDM - Identity Manager
 
-Project Management for the IDM Project
+The Identity Manager - IDM for short - is a unified digital wallet based on open-standards that aims to support multiple decentralized identities.
 
-[![#ipfs-identity](https://img.shields.io/badge/irc-%23ipfs--identity-brightgreen.svg)](https://webchat.freenode.net/?channels=ipfs-identity)
+## Documents
 
-## Current status
+Here is a list of some useful documents. If you're interested in collaborating, please check [Contributing](#contributing).
 
-[![Waffle.io - Columns and their card count](https://badge.waffle.io/ipfs-shipyard/pm-idm.svg?columns=all)](https://waffle.io/ipfs-shipyard/pm-idm)
+[💡 Concept](docs/idm-concept.md)
+[🛠 IDM Breakdown](https://docs.google.com/document/d/1g0TjSPjEM4pryPwJTGhIeE4DBsj-VJpz_JqbfllJUgA)
+[🗓 IDM Workplan](https://docs.google.com/spreadsheets/d/1Venqgkcao2Lcje0mkCxr9u0H037aC3H5IxMfsoaeMoE)
+
+## Team
+- [André Cruz](https://github.com/satazor) - Software engineer
+- [André Sousa](https://github.com/andreforsousa) - Designer
+- [Marco Oliveira](https://github.com/marcooliveira) - Product engineer
+- [Paulo Marcos](https://github.com/paulobmarcos) - Software engineer
+- [Pedro Santos](https://github.com/PedroMiguelSS) - Software engineer
+- [Pedro Teixeira](https://github.com/pgte) - Adviser
+
+## Milestones & Progress
+
+Our milestones are continually updated and detailed in the [IDM Workplan Document](https://docs.google.com/spreadsheets/d/1Venqgkcao2Lcje0mkCxr9u0H037aC3H5IxMfsoaeMoE).
+
+On the first monday of every sprint we will have a remote call, at 4:00 PM GMT, to report the project's current progress.
+If you care to join us you can do so by following this [link]().
```

Comment:
I would remove `If you care to join us you can do so by following this [link]().` and replace with something like: `We create an issue with the label [`progress-call`](https://github.com/ipfs-shipyard/pm-idm/labels/progress-call) for each scheduled call, containing a link to the agenda & notes and with instructions to join to video call`.